docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     image: semtech/mu-identifier:1.10.1
     environment:
       DEFAULT_MU_AUTH_ALLOWED_GROUPS_HEADER: '[{"variables":[],"name":"public"},{"variables":[],"name":"clean"}]'
+      DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_HEADER: "*"
```

Comment:
Do we want to do this on server override instead?

Reply:
I don't think there's anything wrong with putting it in the default docker compose given that Themis is supposed to be a public application. As in, there shouldn't be a Themis deploy where it is not expected to have CORS enabled like this, so this option should be set as the default.

Just my opinion though.

Reply:
without looking into it, how do we enforce that you can't make insert or delete queries?
I see `SPARQL_UPDATE: "true"` on virtuoso. Does that mean it's allowed?

Reply:
The mu-auth config enforces that.

Reply:
ah yes ofc :man_facepalming: In my mind I was comparing this to a sparql connected to a port so I didn't think about mu-auth. should be fine then to have this enabled. Was thinking you could maybe circumvent the frontend-app by doing it, but I was incorrect.